tailwind.config.js

```diff
@@ -1,10 +1,58 @@
 module.exports = {
-  content: [
-    "./index.html",
-    "./src/**/*.{elm,js,ts,jsx,tsx}",
-  ],
+  mode: "jit",
+  content: ["./index.html", "./src/**/*.{html,elm,js,ts,jsx,tsx}"],
+  plugins: [require("daisyui")],
   theme: {
-    extend: {},
+    extend: {
+      height: {
+        screenWithoutNav: 'calc(100vh - 64px)',
+      },
+    },
   },
-  plugins: [],
-}
\ No newline at end of file
+  daisyui: {
+    styled: true,
+    themes: [
```

Comment:
Using the same theme from [webnative-app-template](https://github.com/fission-codes/webnative-app-template/blob/main/tailwind.config.cjs#L7) for consistency